classes/Terminal.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Altapay module for Prestashop
+ *
+ * Copyright © 2020 Altapay. All rights reserved.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+class Terminal extends ObjectModel
+{
+    public $id_terminal;
+    public $display_name;
+    public $remote_name;
+    public $payment_type;
+    public $currency;
+    public $ccTokenControl;
+    public $icon_filename;
+
+    /** @var boolean Enabled or disabled */
```

Comment:
Use the short type for scalar types as that is the one that is consistently allowed
```suggestion
    /** @var bool Enabled or disabled */
```